Fenced_Frames_Ads_Reporting.md

```diff
@@ -51,7 +51,7 @@ Fenced frames can invoke the `reportEvent` API to tell the browser to send a bea
 
 This API is available from same-origin frames within the initial rendered ad document and across subsequent same-origin navigations, but it's no longer available after cross-origin navigations or in cross-origin subframes. (For this API, for chains of redirects, the requestor is considered same-origin with the target only if it is same-origin with all redirect URLs in the chain.) This way, the ad may redirect itself without losing access to reporting, but other sites can't send spurious reports.
 
-The browser processes the beacon by sending an HTTP POST request, like the existing [navigator.sendBeacon](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon).
+The browser processes the beacon by sending an HTTP POST request, like the existing [navigator.sendBeacon](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon). The POST is sent immediately but asynchronously to avoid blocking other actions.
```

Comment:
To be more correct, I'd suggest:
"The POST request is sent immediately or as soon as the corresponding registerAdBeacon is invoked, and is done asynchronously to avoid blocking other actions."

Reply:
Ahhh OK, I think the clarification you're adding here is that, for FF triggered events, the beacon must be invoked, at which point it is sent immediately and etc...jah, thanks, I'll change.

Reply:
Isaac, do you think you'll have a chance to make the update that Shivani requested?

Reply:
Isaac, friendly ping again.